docs/source/conf.py

```diff
@@ -20,6 +20,7 @@
 import sys
 
 import pt_lightning_sphinx_theme
+import pytorch_lightning  # noqa: E402
```

Comment:
this won't work unless we set env variable `PYTHONPATH` to the project root

Reply:
Right!
I wonder if `isort` in `pre-commit` is really using `pyproject.toml` ??

Reply:
mind check it, pls 
cc: @akihironitta

Reply:
@Borda Looking into it :]

Reply:
@arnaudgelas As far as I confirmed locally, `pre-commit` of `isort` uses `pyproject.toml`. However, `pre-commit` only checks changed files, so if you don't add any changes to files with unsorted import statements, `pre-commit` will just ignore them, I suppose.

Reply:
@akihironitta I have also made some checks on my side, and I also confirm that `pyproject.toml`. Note that to force `pre-commit` to run on all files with `isort` hook, you can do `pre-commit run isort --all-files`

Reply:
@Borda @akihironitta If we do not want this line to be moved every time we run isort, I suppose we should skip this line. Right?

Reply:
@arnaudgelas Yes, I think, so shall we add the file to the list in [pyproject.toml](https://github.com/PyTorchLightning/pytorch-lightning/blob/4c6f36e6e14a5e3bace1fe32505ae0fe6f8bc682/pyproject.toml#L25) to skip the isort check on the file?